Source/ASTableView.mm

```diff
@@ -1678,20 +1678,14 @@ - (BOOL)dataControllerShouldSerializeNodeCreation:(ASDataController *)dataContro
 
 - (BOOL)dataController:(ASDataController *)dataController shouldSynchronouslyProcessChangeSet:(_ASHierarchyChangeSet *)changeSet
 {
-  if (ASActivateExperimentalFeature(ASExperimentalNewDefaultCellLayoutMode)) {
-    // Reload data is expensive, don't block main while doing so.
-    if (changeSet.includesReloadData) {
```

Comment:
Why was this removed?

Reply:
This code was not there when the new layout changes was introduced. This is added when we created the experiment afterward. I checked with @nguyenhuy that we did not experiment without this block of code. I will put it back.